### DISCUSSION TITLE
Support /api in src folder ?

### DISCUSSION THREAD

**Discussion by revskill10** [vercel/vercel#4324] [General]
Could we put `/api` folder inside `/src` folder ? In my case, it could be convenient to `import` lib files because my code is inside `src` folder, so instead of `import x from '../src/lib/x'`, i could do `import x from '../lib/x'`

**Comment by idjevm** [comment]
You can actually use routes for your configurations. By default, it looks under /api but if you add a new route in your configuration `vercel.json` you should be able to use the api on any specified folder. 

Check how to configure routes here -> https://vercel.com/docs/configuration#introduction/configuration-reference

**Reply by ctrlplusb** [reply]
I think you mean the [`functions`](https://vercel.com/docs/configuration#project/functions) configuration option right? Specifically the `includeFiles` option within it.

**Comment by jakobrosenberg** [comment]
Did anyone manage to get serverless functions working outside the `api` folder?

I'm wondering about this warning
```
WARNING: It is currently not possible to use the functions property when not using the /api directory for your Serverless Functions.
```
